deploy/criticalresource/apiservice.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apiregistration.k8s.io/v1
+kind: APIService
+metadata:
+  name: v1.admission.apiserver.openshift.io
+spec:
+  group: admission.apiserver.openshift.io
+  version: v1
+  service:
+    name: criticalresource-admissionwebhook
+    namespace: openshift-criticalresource
```

Comment:
Do we need to introduce a `openshift-` namespace here? I'd prefer not to in this repo if we can possibly avoid it.

Reply:
How about open-cluster-management-criticalresource?

Reply:
I understand now why you used an `openshift-` namespace in this prototype, we should think carefully about whether we would ship this with that ns at this point, would the resource live in ocm/api, etc.